src/java/com/zimbra/cs/nginx/NginxLookupExtension.java

```diff
@@ -1046,7 +1046,9 @@ private void search(NginxLookupRequest req) throws NginxLookupException {
                         if (doDnsLookup) {
                             upstreamHost = this.getIPByIPMode(upstreamHost).getHostAddress();
                         }
-                        String upstreamPort = req.proto.equals(IMAP) ? imapServer.getImapBindPortAsString() : imapServer.getImapSSLBindPortAsString();
+                        String upstreamPort = req.proto.equals(IMAP) ?
```

Comment:
Probably a dumb question but what happens when using the mailboxd hosted IMAP?  Does that not go through here and if it does, which port number should be used?